node/libs/crypto/src/ed25519/tests.rs

```diff
@@ -0,0 +1,56 @@
+#[test]
+fn test_ed25519() -> Result<(), Box<dyn std::error::Error>> {
+    use ed25519_dalek::{
```

Comment:
move "use" clause to the top of the file.

Reply:
Then there will be some warnings that imports are not used.